gulpfile.babel.js

```diff
@@ -90,7 +90,10 @@ proGulp.task("buildAllScripts", (() => {
         },
         plugins: [
             new webpack.DefinePlugin({
-                "process.env.NODE_ENV": JSON.stringify(NODE_ENV)
```

Comment:
Not sure it's correct. Shouldn't it be:
```js
new webpack.DefinePlugin({
    "process.env.NODE_ENV": JSON.stringify(NODE_ENV),
    "process.env.EXEC_ENV": JSON.stringify(EXEC_ENV)
})
```
I'm not sure the two things are equivalent. (https://github.com/webpack/docs/wiki/list-of-plugins#defineplugin)